Runtime/AvatarCreator/Scripts/Managers/AuthManager.cs

```diff
@@ -40,23 +41,23 @@ public static void SetUser(UserSession session)
             OnSignedIn?.Invoke(userSession);
         }
 
-        public static async void SendEmailCode(string email)
+        public static async Task SendEmailCode(string email, CancellationToken cancellationToken = default)
         {
             try
             {
-                await AuthAPIRequests.SendCodeToEmail(email, userSession.Id);
+                await AuthAPIRequests.SendCodeToEmail(email, userSession.Id, cancellationToken);
```

Comment:
I think this cancel token is not necessary

Reply:
Right now, there's no token to pass separately to AuthAPI.